sessions/react-state/demo-food-order-end/src/App.js

```diff
@@ -11,19 +11,65 @@ export default function App() {
 }
 
 function FoodOrder() {
-  const [food, setFood] = useState("❓");
+  const [food, setFood] = useState("?");
   const [vegan, setVegan] = useState(true);
 
   return (
     <>
       <div>
-        <button onClick={() => setFood("🍔")}>🍔</button>
-        <button onClick={() => setFood("🍕")}>🍕</button>
-        <button onClick={() => setFood("🌮")}>🌮</button>
+        <button
+          onClick={() =>
+            setFood(
+              <span role="img" aria-label="A hamburger">
+                🍔
+              </span>
+            )
+          }
+        >
+          <span role="img" aria-label="A hamburger">
+            🍔
+          </span>
+        </button>
+        <button
+          onClick={() =>
+            setFood(
+              <span role="img" aria-label="A pizza">
+                🍕
+              </span>
+            )
+          }
+        >
+          <span role="img" aria-label="A pizza">
+            🍕
+          </span>
+        </button>
+        <button
+          onClick={() =>
+            setFood(
+              <span role="img" aria-label="A taco">
+                🌮
+              </span>
+            )
+          }
+        >
+          <span role="img" aria-label="A taco">
+            🌮
+          </span>
+        </button>
       </div>
+
       <p>Selected food: {food}</p>
```

Comment:
We could to something like this, but the label is a bit non-sense. We'd need to establish a more complex or second state to store a label  for the food :-/

```suggestion
      <p>Selected food: <span role="img" aria-label="Food choice">{food}</span></p>
```

Reply:
I agree, we should leave it the current way and discuss the issue with the students if it comes up.

Reply:
ok from @djfarly -> resolved